### PR TITLE
Added basic product review functionality

### PR DESCRIPTION
**Pull request by subhadeepdas91**
I really think that product review feature is required. 
As a stepping stone to implement product review I suggest this merge request
related issue #3513

### Pull Request Checklist

1. [ ] Privileged views and APIs are guarded by proper permission checks.
1. [ ] All visible strings are translated with proper context.
1. [ ] All data-formatting is locale-aware (dates, numbers, and so on).
1. [ ] Database queries are optimized and the number of queries is constant.
1. [X] Database migration files are up to date.
1. [ ] The changes are tested.
1. [ ] The code is documented (docstrings, project documentation).
1. [ ] GraphQL schema and type definitions are up to date.
1. [ ] Changes are mentioned in the changelog.

### CODE REVIEW

File: saleor/product_review/models.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+from django.core.exceptions import ValidationError
+from django.core.validators import MinValueValidator, MaxValueValidator
+from django.db import models
+
+from saleor.product.models import Product
+
+
+class ProductReview(models.Model):
+    product = models.ForeignKey(Product,related_name='products',on_delete=models.CASCADE)
+    user = models.ForeignKey(settings.AUTH_USER_MODEL,on_delete=models.CASCADE)
```
Comment (bh) [inline saleor/product_review/models.py]: missing whitespace

File: saleor/product_review/models.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    score = models.FloatField(
+        default=settings.PRODUCT_REVIEW_MIN_SCORE,
+        validators=[MinValueValidator(settings.PRODUCT_REVIEW_MIN_SCORE),MaxValueValidator(settings.PRODUCT_REVIEW_MAX_SCORE)]
```
Comment (bh) [inline saleor/product_review/models.py]: missing whitespace

**Comment by maarcingebala** [comment]
Thanks for opening a PR! In my opinion however it's not ready to merge yet. Before we do that, this needs a proper discussion and planning e.g.
- what models and model fields do we exactly need?
- should we support reviews moderation in the dashboard?
- rendering reviews in the storefront

Also this whole PR should ideally provide full backend implementation which also means adding GraphQL API for that. @subhadeepdas91 Do you think you'd be able to work on that? If not, right now I'm rather against merging partial PRs.

**Comment by maarcingebala** [comment]
Closing for now. If we want to continue, let me know through this PR or issue #3513.

**Pull request closed without merging**